src/apiCall.js

```diff
@@ -47,139 +51,150 @@ module.exports = function apiCall(file, apiPort) {
   const swaggerParserd = config.apiCalls.swagger
     .map(fileEvaluator)
 
-  swaggerParserd.forEach((_req) => {
-    const itMethod = getItFunction(_req);
-
-    itMethod(_req.name || _req.call, async function itFn() {
-      const req = evaluateFaker(_req, fakerScopes.test)
-      this.timeout(apiPort.get('TIMEOUT'))
-
-      if (!operationIds[req.call]) {
-        throw new Error(`No swagger operation exists with operationId "${req.call}"`)
-      }
+  swaggerParserd.forEach((reqParams) => {
+    const repeat = reqParams.repeat || 1
+    const requests = _.times(repeat)
+      .map(() => _.clone(reqParams))
 
-      const params = req.parameters || ''
-      if (params) {
-        apiPort.validateParams(operations[req.call].parameters || [], params)
-      }
+    requests.forEach((req, i) => {
+      const name = getItName(req)
+      req.itText = repeat === 1 ? name : `${name} - ${i}`
+    })
 
-      let allReqData = {}
-      const reqData = req.data || {}
-      let basicAuth = req.basicAuth || {}
-      if (basicAuth.$file) {
-        basicAuth = apiPort.getDataFromFile(basicAuth.$file, file)
-        basicAuth = apiPort.resolveObject(basicAuth)
-        delete basicAuth.$file
-      }
+    requests.forEach((_req) => {
```

Comment:
because of this new loop, most of the change fro here are indentation only, I would advice to review with  `Hide whitespace changes` enabled.  Here is a shortcut to it :) https://github.com/testbetter/openapitest/pull/13/files?w=1